x11-packages/libxml++-5.0/build.sh

```diff
@@ -0,0 +1,9 @@
+TERMUX_PKG_HOMEPAGE=https://libxmlplusplus.github.io/libxmlplusplus/
+TERMUX_PKG_DESCRIPTION="A C++ wrapper for the libxml2 XML parser library (version 5.0)"
+TERMUX_PKG_LICENSE="LGPL-2.1"
+TERMUX_PKG_MAINTAINER="@IntinteDAO"
+_MAJOR_VERSION=5.0
+TERMUX_PKG_VERSION=${_MAJOR_VERSION}.4
+TERMUX_PKG_SRCURL=https://download.gnome.org/sources/libxml++/${_MAJOR_VERSION}/libxml++-${TERMUX_PKG_VERSION}.tar.xz
```

Comment:
We're trying to reduce the use of this type of global helper variable.
Please use parameter expansion instead.
```suggestion
TERMUX_PKG_VERSION=5.0.4
TERMUX_PKG_SRCURL=https://download.gnome.org/sources/libxml++/${TERMUX_PKG_VERSION%.*}/libxml++-${TERMUX_PKG_VERSION}.tar.xz
```

Reply:
Termux `libxml++2.6` does that that's why I didn't mention it

https://github.com/termux/termux-packages/blob/1a6e09b8db7ba34029a4c25e7af9ff9306aeb16f/x11-packages/libxml%2B%2B-2.6/build.sh#L5

Should this PR also change `libxml++2.6`?

Reply:
Should be fixed :)